examples/quickstart/jupyter-notebooks/druid-visualization.ipynb

```diff
@@ -0,0 +1,620 @@
+    "with your Druid data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07a16047-dba1-4f45-9fcb-6130c37eca61",
+   "metadata": {},
+   "source": [
+    "## Display data with a bar graph\n",
+    "\n",
+    "Tabular format is OK for scanning data, however you can use visualization and plotting tools that work with Jupyter notebooks to visualize data as plots or graphs. This section uses [Bokeh](https://bokeh.org/) to illustrate some basic plots using Druid data.\n",
+    "\n",
+    "In this section, you create a simple bar chart showing the channels with the most additions during the time range."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8420df27-0e80-48d1-a806-43f6bf81dd16",
+   "metadata": {},
+   "source": [
+    "First, call `output_notebook()` to set Bokeh to output plots inline within the notebook. This lets you view your plot inline. You can also use Bokeh's `output_file` function to write your plot to an HTML file. If you want to experiment with `output_file` add it to the list of imports."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d824a82f-9e71-43c4-b289-78d4970c5cfc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_notebook()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5859d743-a5a7-445c-a06f-89b8c30856ed",
+   "metadata": {},
+   "source": [
+    "There are several ways to use Bokeh with a DataFrame. In this case, make a list of channels to serve as the x axis of our plot. For the y axis,  divide the total additions by 100000 for ease of display."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70b40bfc-106d-462c-a5a1-51d0da4c4498",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "channels = additions_df.channel.to_list()\n",
+    "total_additions = [x / 100000 for x in additions_df.additions.to_list()]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "efc5f142-8b68-4430-a666-2ed961d732b9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "channels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f619fd3-6a01-4dc6-b8eb-81e4b4ae1a31",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "total_additions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6b430b4c-df81-419d-a81e-63bb0150031b",
+   "metadata": {},
+   "source": [
+    "Next, initialize your Bokeh plot (figure) with some basic configurations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "07644ccd-1f0c-4a50-8929-82559f9bf803",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    " # Create a new plot with a title. Set the size in pixels\n",
+    "bar_plot = figure(height=500, width=750, x_range=channels,  title=\"100000 x Additions\",\n",
```

Comment:
I think the label `100000 x Additions` more accurately reflects the y-axis, and the title should be something more descriptive like "number of additions by channel" or something